src/components/TextField/tests/TextField.test.tsx

```diff
@@ -812,4 +812,69 @@ describe('<TextField />', () => {
       expect(textField.find(Connected).prop('right')).toEqual(connectedRight);
     });
   });
+
+  describe('clearButton', () => {
+    it('adds a clear button that clears the input value', () => {
+      const spy = jest.fn();
+      const textField = mountWithAppProvider(
+        <TextField 
+          id="MyTextField"
+          label="TextField"
+          onChange={spy}
+          clearButton
+        />
+      )
+      textField.find('.ClearButton').simulate('click');
```

Comment:
Officially we don't use test IDs anymore either but I feel like here it might still make more sense than using the class in this case.